py/serial_keel.py

```diff
@@ -197,18 +197,25 @@ async def _read(self) -> None:
 
         while True:
             response = await self.skws.read()
+
+            # When we get messages, they may come utf-8 encoded. Those are not very nice
+            # to read. Try to decode them and replace the original stream
+            try:
+                message_ref = response.get('Ok', {}).get('Async', {}).get('Message', {})
+                decoded_message = bytes(message_ref['message']).decode('utf-8')
+                message_ref['message'] = decoded_message.strip()
```

Comment:
Hmm, just noting that using `.strip()` here is good for log readability, but this also means the Py client will not allow users to receive messages with surrounding whitespace.
I don't think it's a problem now, just making the note.